Back-End/controllers/students_controllers/students_controllers.js

```diff
@@ -0,0 +1,62 @@
+const db = require('../../db');
+
+const courses_s = (req, res) => {
```

Comment:
What is this `_s` stand for? students? You will have to rename the functions correctly and follow the JavaScript function naming conventions,

From the query, it looks like you are retrieving all the courses so a name like `getAllCourses` is better suited